oclc.py

```diff
@@ -677,565 +665,6 @@ def graphs_for(self, identifier):
         self.log.debug("END GRAPHS FOR %r", identifier)
 
 
-class OCLCClassifyAPI(object):
```

Comment:
Moved all of this to `oclc_classify.py` since it's specific to the Classify API.